File: payments/search/nodejs/basic.js
var MP = require ("mercadopago");

var mp = new MP ("CLIENT_ID", "CLIENT_SECRET");

var filters = {
	"status": "approved",
	"offset": 0,
	"limit": 10
};

mp.searchPayment (filters, function (err, data){
	if (err) {
		console.log (err);
	} else {
		console.log (JSON.stringify (data, null, 4));
	}
});

File: payments/search/python/basic.py
# coding: UTF-8

import os, sys
import mercadopago

import json

def index(req, **kwargs):
    mp = mercadopago.MP("CLIENT_ID", "CLIENT_SECRET")
    
    filters = {
        "status": "approved",
        "offset": 0,
        "limit": 10
    }

    searchResult = mp.search_payment(filters)

    return json.dumps(searchResult, indent=4)